pkgs/development/python-modules/django-allauth/default.nix

```diff
@@ -52,6 +56,8 @@ buildPythonPackage rec {
     requests-oauthlib
   ] ++ pyjwt.optional-dependencies.crypto;
 
+  preBuild = "python -m django compilemessages";
```

Comment:
This should use python.interpreter to work for pypy etc

Reply:
please view https://github.com/NixOS/nixpkgs/pull/312003